From 0bfc9be6575a7bb0a09558b56577fe03fd952279 Mon Sep 17 00:00:00 2001
From: Tomas Zak <tomas.zak@nic.cz>
Date: Thu, 9 Apr 2026 19:24:48 +0200
Subject: [PATCH] tools: compile ccache package before liblzo package

Liblzo package was compiled before ccache package which caused
compilation error
---
 tools/Makefile | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/tools/Makefile b/tools/Makefile
index 79d3954fef..c63a3db84e 100644
--- a/tools/Makefile
+++ b/tools/Makefile
@@ -235,6 +235,9 @@ $(curdir)/ := .config prereq
 
 $(curdir)/install: $(curdir)/compile
 
+# compile ccache before liblzo
+$(curdir)/liblzo/compile := $(curdir)/ccache/compile
+
 tools_enabled = $(foreach tool,$(sort $(tools-y) $(tools-)),$(if $(filter $(tool),$(tools-y)),y,n))
 $(eval $(call stampfile,$(curdir),tools,compile,,_$(subst $(space),,$(tools_enabled)),$(STAGING_DIR_HOST)))
 $(eval $(call stampfile,$(curdir),tools,check,$(TMP_DIR)/.build,,$(STAGING_DIR_HOST)))
-- 
2.47.3

